install/kubernetes/cilium/templates/cilium-preflight/clusterrole.yaml

```diff
@@ -72,29 +70,51 @@ rules:
 - apiGroups:
   - cilium.io
   resources:
-  - ciliumnetworkpolicies
-  - ciliumnetworkpolicies/status
+  - ciliumbgploadbalancerippools
+  - ciliumbgppeeringpolicies
+  - ciliumclusterwideenvoyconfigs
   - ciliumclusterwidenetworkpolicies
-  - ciliumclusterwidenetworkpolicies/status
+  - ciliumegressnatpolicies
   - ciliumendpoints
-  - ciliumendpoints/status
-  - ciliumnodes
-  - ciliumnodes/status
+  - ciliumendpointslices
+  - ciliumenvoyconfigs
   - ciliumidentities
   - ciliumlocalredirectpolicies
-  - ciliumlocalredirectpolicies/status
-  - ciliumegressnatpolicies
-  - ciliumendpointslices
-  - ciliumbgppeeringpolicies
-  - ciliumbgploadbalancerippools
+  - ciliumnetworkpolicies
+  - ciliumnodes
```

Comment:
Does preflight need to list all of these resources? (or create, etc.)

Reply:
It probably doesn't but unfortunately we need to keep those two files in sync otherwise if we add or remove permissions in the cilium-agent we will forget to add/remove on this file.